packages/runtime/runtime-definitions/src/dataStoreContext.ts

```diff
@@ -71,6 +69,8 @@ export interface IContainerRuntimeBase extends
     readonly logger: ITelemetryBaseLogger;
     readonly clientDetails: IClientDetails;
 
+    getEntryPoint(): Promise<IFluidObject>;
```

Comment:
I'd think that the only place it should show up is IRuntime, no? It's purpose is communicating up (to loader), not down (to data stores).

Reply:
you're probably right that this didn't need to push down to here. We'll need the same thing for the datastore runtime as well